src/main/frontend/config.cljs

```diff
@@ -374,12 +374,21 @@
   [repo-url relative-path]
   (when (and repo-url relative-path)
     (cond
-      (and (or (util/electron?) (mobile-util/is-native-platform?)) (local-db? repo-url))
+      (and (or (util/electron?) (mobile-util/native-android?)) (local-db? repo-url))
       (let [dir (get-repo-dir repo-url)]
         (if (string/starts-with? relative-path dir)
           relative-path
           (str dir "/"
                (string/replace relative-path #"^/" ""))))
+
+      (and (mobile-util/native-ios?) (local-db? repo-url))
+      (let [dir (-> (get-repo-dir repo-url)
+                    (string/replace "file:///" "file:/")
+                    (string/replace "%20" " "))]
```

Comment:
😆 can bereplaced by decodeURI

Reply:
The full dir here is fixed: `"file:///private/var/mobile/Library/Mobile%20 Documents/iCloud~com~logseq~logseq/Documents/T1/logseq/"`, so `(string/replace % "%20" " ")` is enough.

Reply:
hmmm. The graph name is not fixed..

Reply:
okay, will try decodeURI here

Reply:
Never mind, %20 is enough for graph names.

Reply:
No need to decode the dir here anymore by encoding the file-path in the database.

The PR is ready for final review.